econml/tests/test_dml.py

```diff
@@ -149,7 +149,7 @@ def test_can_use_vectors(self):
     def test_can_use_sample_weights(self):
         """Test that we can pass sample weights to an estimator."""
         dmls = [
-            LinearDMLCateEstimator(LinearRegression(), LinearRegression(), featurizer=FunctionTransformer()),
+            LinearDMLCateEstimator(LinearRegression(), 'auto', featurizer=FunctionTransformer()),
```

Comment:
Can we do 'auto' for both here, so that we test that both our auto options can accept sample_weights?

Reply:
Done!